src/apsbits/utils/metadata.py

```diff
@@ -80,18 +80,14 @@ def get_md_path(iconfig: Optional[dict[str, Any]] = None) -> Optional[str]:
     return str(path)
 
 
-def re_metadata(
-    iconfig: Optional[dict[str, Any]] = None, cat: Optional[Any] = None
-) -> dict[str, Any]:
+def re_metadata(iconfig: Optional[dict[str, Any]] = None) -> dict[str, Any]:
```

Comment:
In `init_RE()`, *iconfig* can has type `Mapping[str, Any]`, but here it needs to be `dict[str, Any]`. Maybe this one should also be `Mapping[str, Any]`? And same for `get_md_path()`.